### PR TITLE
[Element 1.x.x] Resolve unhandled rejection on Windows and Ubuntu

### PR DESCRIPTION
**Pull request by honglax**
Resolve this issue [**FLO-2208**](https://tricentisflood.atlassian.net/browse/FLO-2208)

Just a little fixing base on the usage of `command-exists` [here](https://www.npmjs.com/package/command-exists).

### CODE REVIEW

File: packages/cli/src/utils/updateCheck.ts [hunk 1]
```diff
@@ -33,9 +40,18 @@ function yarnUpdateMessage(
...
+		if (err) {
+			console.error(err)
+			result = undefined
+		} else {
+			result = isYarnExists
+				? chalk`Get it by running {greenBright yarn global upgrade element-cli@${distTag}}`
+				: undefined
+		}
+	})
+	return result
```
Comment (nick-flood) [inline packages/cli/src/utils/updateCheck.ts:54]: is commandExists asynchronous? Looks like it from the callback, won't we get order of execution errors?

**Review by nick-flood — CHANGES_REQUESTED** [review]

**Comment by nick-flood** [comment]
Rather than using the async version and having to re-write the function signature i think we should use the sync version `commandExists.sync`

**Comment by honglax** [comment]
> Rather than using the async version and having to re-write the function signature i think we should use the sync version `commandExists.sync`

I'll give it a try.

**Comment by honglax** [comment]
> Rather than using the async version and having to re-write the function signature i think we should use the sync version `commandExists.sync`

Hi @nick-flood, I've changed to the sync version and it works well. Please re-review this PR. Thank you.

File: packages/cli/src/utils/updateCheck.ts (hunk 1 above)
Comment (honglax) [reply packages/cli/src/utils/updateCheck.ts]: Hi @nick-flood, I've changed to the sync version and it works well. Please re-review this PR. Thank you.

**Review by phileba — APPROVED** [review]

**Review by HieuLeKMS — APPROVED** [review]

**Review by nick-flood — APPROVED** [review]

**Pull request merged**